blocks/editable/format-toolbar/index.js

```diff
@@ -19,22 +19,22 @@ const { ESCAPE, LEFT, RIGHT, UP, DOWN } = keycodes;
 const FORMATTING_CONTROLS = [
 	{
 		icon: 'editor-bold',
-		title: __( 'Bold' ),
+		title: __( 'Bold — ⌘B' ),
```

Comment:
Should we use `${ __( 'Bold') } — ⌘B` instead? As we are not expecting keyboard shortcuts changed during translation, this way translations for 'Bold' get reused.

Reply:
Good point, though I'm not sure dash and order should not be translatable. Maybe we can just omit the shortcut itself from the translation.

Reply:
> Should we use ${ __( 'Bold') } — ⌘B instead? 

Alas, this wouldn't work for RTL languages like Hebrew and Arabic.

Reply:
I think it would be safe to use something like ${ __( 'Bold' ) } ${ __( '-' ) }   ⌘B. This way translations are reused and we address the point raised by @iseulde.
Regarding the RTL point, It's a nice question, honestly, I don't know if in RTL normally the shortcuts appear inverted, but in LTR, if the shortcuts appeared like "⌘B — Bold" it would be ok so maybe in RTL both ways are also "ok". To be safe be on the safe side we can use the original version, I don't have a strong opinion here.